xbmc/guilib/GUIWindowManager.cpp

```diff
@@ -699,9 +703,9 @@ void CGUIWindowManager::PreviousWindow()
     }
     return;
   }
-  m_windowHistory.pop();
+  m_windowHistory.pop_back();
   int previousWindow = GetActiveWindow();
-  m_windowHistory.push(currentWindow);
+  m_windowHistory.push_back(currentWindow);
```

Comment:
Please use `emplace_back` (C++11), which is always to prefer over `push_back`.

Reply:
done